sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/FeedResponse.java

```diff
@@ -7,6 +7,7 @@
 import com.azure.core.util.paging.ContinuablePage;
 import com.azure.cosmos.BridgeInternal;
 import com.azure.cosmos.CosmosDiagnostics;
+import com.azure.cosmos.implementation.ClientSideRequestStatistics;
```

Comment:
is this needed?(there is no other changes in this file except the import )

Reply:
Fixed